ribs/archives/_grid_archive.py

```diff
@@ -155,6 +155,55 @@ def boundaries(self):
         """
         return self._boundaries
 
+    def retessellate(self, new_dims):
```

Comment:
Can you put this method in a separate PR where you also write some tests for it? Thanks!